docs/cli/init.mdx

```diff
@@ -8,9 +8,42 @@ Places the Service Worker file into a given public directory of your application
 ## Usage
 
 ```bash
-$ npx msw init <PUBLIC_DIR>
+$ npx msw init <PUBLIC_DIR> [options]
 ```
 
+### Positionals
+
+#### `PUBLIC_DIR`
```

Comment:
I don't know what a positional is 😅
Perhaps, because we only got 1, we could just use this as heading?

```suggestion
### `PUBLIC_DIR`
```